examples/private_channels.rb

```diff
@@ -0,0 +1,51 @@
+require 'sinatra'
+require 'sinatra/cookies'
+require 'sinatra/json'
+require 'pusher'
+
+
+# You can get these variables from http://dashboard.pusher.com
+pusher = Pusher::Client.new(
+  app_id: 'YOUR-APP-ID',
+  key: 'YOUR-APP-KEY',
+  secret: 'YOUR-APP-SECRE`t',
+  cluster: 'CLUSTER'
+)
+
+# Emulate rails behaviour where this information would be stored in session
+get '/signin' do
+  cookies[:user_id] = 'example_cookie'
+  'Ok'
+end
+
+# Auth endpoint: https://pusher.com/docs/channels/server_api/authenticating-users
+post '/pusher/auth' do
+  channel_data = {
+      user_id: 'example_user',
+      user_info: {
+        name: 'example_name',
+        email: 'example_email'
+      }
+    }
+
+  if cookies[:user_id] == 'example_cookie'
+    response = pusher.authenticate(params[:channel_name], params[:socket_id], channel_data)
+    json response
+  else
+    status 403
+  end
+end
+
+get '/pusher_trigger' do
```

Comment:
I am not sure if we need this. I guess we either leave this and add html page that listens on `private-channel-test` so the example will be complete, or just leave it as an Authentication example.